specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -103,6 +103,39 @@
         }
       }
     },
+    "/providers/Microsoft.ResourceHealth/emergingIssues": {
+      "get": {
+        "tags": [
+          "EmergingIssues"
+        ],
+        "operationId": "EmergingIssues_Get",
```

Comment:
Not list?  "Get" should be used for single resource retrievals

Reply:
Yes. It is a single resource. As far as we see it, it is just 'emergingIssue'.

Reply:
The RPC treats resource type GETs as collections.  This is a collection GET of all emergingIssues and needs to follow that pattern.  If you want a singleton it should be emergingIssues/default and the GET emergingIssues API will return a collection with only that resource.

Reply:
You may want to rethink exposing all this info as one single resource though.  What if there are hundreds of issues?  Do you really want all that info in one resource?  The payload may get very large.

Reply:
No. this payload won't get large. This emerging issue is only used for big outage event shows up on status.azure.com. We don't allow user to publish hundreds of issues. One single object is good enough. 
Are you suggesting that since I'm using 'EmergingIssues' instead of 'EmergingIssue', I need to return a collection. What if I make it to be singular, can I just use 'GET' and return a object? I see our metadata API is doing that. If this doesn't work, I'll consider to keep as is, but the collection will always have one item in it. I don't quite get '/default' you are suggesting, any example I can follow?

Reply:
It should be "emergingIssues" because it is in a URI segment that represents a resource type (and that should be the name of the resource type).  There are a lot of examples of the singleton pattern.  One is:  https://docs.microsoft.com/en-us/rest/api/sql/server%20auditing%20settings/listbyserver

Reply:
done the suggested change.